### PR TITLE
Add redlines styles

### PR DESCRIPTION
**Pull request by franji**
# Type of change? 

- [ ] Bug fix
- [ ] New feature
- [x] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 
Applied style guidelines.

**Checklist:**

- [ x] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: src/components/pages/devices/flyouts/SIMManagement/SIMManagement.scss [hunk 1]
```diff
@@ -15,14 +15,20 @@
...
     align-items: center;
-    display: flex;
     justify-content: space-between;
     width: 100%;
+    border-bottom-style: solid;
+    border-bottom-width: 1px;
+    padding-bottom: 44px;
+  }
+
+  .btn-toolbar {
```
Comment (spryor) [inline src/components/pages/devices/flyouts/SIMManagement/SIMManagement.scss]: stylistic nit: `.btn-toolbar { padding-top: 44px; }` [see styleguide](https://github.com/Azure/pcs-remote-monitoring-webui/wiki/Code-Guide#css) - Single vs. Multiple Declarations

Also `@include rem-fallback(padding-top, 44px)`

File: src/components/pages/devices/flyouts/SIMManagement/SIMManagement.js [hunk 2]
```diff
@@ -74,6 +82,12 @@ export class SIMManagement extends LinkedComponent {
               </Section.Content>
             </Section.Container>
           }
+          <BtnToolbar>
+            {!isPending && <Btn primary={true} disabled={!provider} onClick={this.showProvider} type="submit">{t('devices.flyouts.new.apply')}</Btn>}
+            {!isPending && <Btn svg={svgs.cancelX} onClick={onClose}>{t('devices.flyouts.new.cancel')}</Btn>}
```
Comment (spryor) [inline src/components/pages/devices/flyouts/SIMManagement/SIMManagement.js]: If both of these buttons have the same functionality might as well put the condition inside to just change the text.

File: src/components/pages/devices/flyouts/SIMManagement/SIMManagement.scss [hunk 3]
```diff
@@ -15,16 +15,20 @@
...
   }
 
   .sim-management-selector {
     align-items: center;
-    display: flex;
     justify-content: space-between;
     width: 100%;
+    border-bottom-style: solid;
+    border-bottom-width: 1px;
+    padding-bottom: 44px;
```
Comment (spryor) [inline src/components/pages/devices/flyouts/SIMManagement/SIMManagement.scss]: Sorry, missed this before. This padding also needs `@include rem-fallback`

File: src/components/pages/devices/flyouts/SIMManagement/SIMManagement.js [hunk 4]
```diff
@@ -5,8 +5,16 @@ import { Trans } from 'react-i18next';
 import { Link } from 'react-router-dom'
 
 import { permissions } from 'services/models';
-import { LinkedComponent } from 'utilities';
-import { FormControl, Protected } from 'components/shared';
+
+import { LinkedComponent, svgs } from 'utilities';
+
+import {
```
Comment (ppathan) [inline src/components/pages/devices/flyouts/SIMManagement/SIMManagement.js]: @mechaffin I had to resolve conflicts. Can you make sure this section is still accurate?

File: src/components/pages/devices/flyouts/SIMManagement/SIMManagement.js [hunk 5]
```diff
@@ -76,6 +86,10 @@ export class SIMManagement extends LinkedComponent {
                 </Section.Content>
               </Section.Container>
             }
+            <BtnToolbar>
+              {!isPending && <Btn primary={true} disabled={!provider} onClick={this.showProvider} type="submit">{t('devices.flyouts.new.apply')}</Btn>}
+              <Btn svg={svgs.cancelX} onClick={onClose}>{isPending ? t('devices.flyouts.new.close') : t('devices.flyouts.new.cancel')}</Btn>
+            </BtnToolbar>
```
Comment (ppathan) [inline src/components/pages/devices/flyouts/SIMManagement/SIMManagement.js]: @mechaffin I had to resolve conflicts. Can you make sure this is still accurate, with `Protected` section?

Comment (mechaffin) [reply src/components/pages/devices/flyouts/SIMManagement/SIMManagement.js]: Yes, this looks correct for the Protected section.

**Pull request merged**